frontend/src/components/Icons/icons.ts

```diff
@@ -0,0 +1,232 @@
+export enum icons {
```

Comment:
Not sure using enums is a good idea, its much simpler to simply use the CamelCase name.

Also, we will have to constantly update this for every new icons.

Reply:
yes, your probably right. You would also need to import the icons enum manually everywhere. I wish there would be a nicer way for an autocomplete here